src/rpc/blockchain.cpp

```diff
@@ -1637,6 +1643,337 @@ static UniValue savemempool(const JSONRPCRequest& request)
+            "                                          \"abort\" for aborting the current scan (returns true when abort was successful)\n"
+            "                                          \"status\" for progress report (in %) of the current scan\n"
+            "2. \"options\"                      (object, optional)\n"
+            "      \"pubkeys\":[\"pubkey\",...]    (array of strings, optional) An array of HEX encoded public keys\n"
+            "      \"addresses\":[\"address\",...] (array of strings, optional) An array of bitcoin addresses\n"
+            "      \"xpubs\":                    (array of xpub objects that will be used to derive child keys with the given lookup window after m/0/k and m/1/k scheme)\n"
+            "           [\n"
+            "               {\n"
+            "                \"xpub\":\"<xpub>\",  (base58check encoded extended public key (xpub)\n"
+            "                \"lookupwindow\": [<startindex>, <stopindex>] (An array with two integers that does define the range of keys that will be deriven for the given xpubs, default is 0 to 1000)\n"
+            "                }\n"
+            "           ]\n"
+            "      \"rawsweep\": {\n             (object, optional) Optionally creates a raw sweep transaction\n"
+            "          \"address\": \"address\",   (string, optional) Address where the funds should be sent to\n"
+            "          \"feerate\": n,           (numeric, optional, default not set: makes wallet determine the fee) Set a specific fee rate in " + CURRENCY_UNIT + "/kB\n"
+            "          \"conf_target\": n,       (numeric, optional) Confirmation target (in blocks), has no effect if feerate is provided\n"
+             "       }\n"
+            "\nResult:\n"
+            "{\n"
+            "  \"unspents\": [\n"
+            "    {\n"
+            "    \"txid\" : \"transactionid\",     (string) The transaction id\n"
+            "    \"vout\": n,                    (numeric) the vout value\n"
+            "    \"scriptPubKey\" : \"script\",    (string) the script key\n"
+            "    \"amount\" : x.xxx,             (numeric) The total amount in " + CURRENCY_UNIT + " received by the address\n"
+            "    \"height\" : n,                 (numeric) Height of the unspent transaction output\n"
+            "   }\n"
+            "   ,...], \n"
+            " \"total_amount\" : x.xxx,          (numeric) The total amount of all found unspent outputs in " + CURRENCY_UNIT + "\n"
+            " \"rawsweep_tx\" : \"value\",       (string) The hex-encoded raw transaction of the optional sweep transaction\n"
+            " \"rawsweep_vsize\" : \"value\",     (numeric) virtual transaction size of the sweep transaction including signatures\n"
+            " \"rawsweep_fee\" : \"value\",       (numeric) Estimated fee for the sweep transaction in " + CURRENCY_UNIT + "\n"
+            "]\n"
```

Comment:
Replace `]` with `}`.

Reply:
Commit "Commit "Blockchain/RPC: Add scantxoutset method to scan UTXO set"

The response is an object, not an array.